docs/getEnergyConsumedSamples.md

```diff
@@ -0,0 +1,42 @@
+# getEnergyConsumedSamples
+
+A quantity sample type that measures the amount of energy consumed.
+
+Permission required:
+
+- `AppleHealthKit.Constants.Permissions.EnergyConsumed`
```

Comment:
I think this permission is wrong, it should be `DietaryEnergy`. When I tested this didn't work. I don't think we have this permission on the enum. Can you check that? How did you tested this and not got permission error?

Reply:
My bad, it seems I have created a mess. It is supposed to be `EnergyConsumed` because that's how Apple renamed the property recently. It used to be called "Dietary Energy" in both Health Apps and their SDK.

In my previous PR I added all missing permissions for Nutrition Data. However this second (`calories-and-protein-consumed`) branch I have created didn't include my changes from the previous branch. 

I am really new to this, I will have to read how to make sure I am having the latest version of the repository I have forked.

So basically, I merged my first branch `fix-read-permissions-for-nutrition` into `calories-and-protein-consumed` (the one in this PR) & everything looks and works great. (Permissions are requested and data is fetched).

How should I proceed? I can create a demo repository to test it yourself? I see that github already attached my merge of branches to this discussion.

Reply:
@Macelai I think this one is resolved, right? :D

Reply:
About this one yeah, but looks like we have some conflicts, can you update this branch with changes required by @lucaspbordignon on the other PR and solve the conflicts, it should be easy to solve them, with you need help, please let me know!